repos/system_upgrade/common/libraries/grub.py

```diff
@@ -70,3 +70,23 @@ def get_grub_device():
     api.current_logger().info('GRUB is installed on {}'.format(grub_dev))
     # if has_grub(grub_dev):
     return grub_dev if has_grub(grub_dev) else None
+
+
+def is_grub_cfg_bls(grub_cfg_msg):
+    """
+    Check if GRUB config file is BLS aware
+    """
+    if 'insmod blscfg' in grub_cfg_msg.grub_cfg:
+        return True
+    return False
+
+
+def is_blscfg_enabled(default_grub_msg):
+    """
+    Check if GRUB_ENABLE_BLSCFG is true
+    """
+    grub_options_lst = default_grub_msg.default_grub_info
+    default_grub_options = {
+        option.name: option.value.strip('"') for option in grub_options_lst
+    }
+    return True if default_grub_options.get('GRUB_ENABLE_BLSCFG', '') == 'true' else False
```

Comment:
we should work with the parsed content, and these two functions seem to be confusing in my POV . Based on their names and descriptions, I am not able to see what is difference between them unless I check the code.